groups/live/templates/bulk_gateway_user_data.tpl

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+# Redirect the user-data output to the console logs
+exec > >(tee /var/log/user-data.log|logger -t user-data -s 2>/dev/console) 2>&1
+
+#Run Ansible playbook for server setup using provided inputs
+echo '${ANSIBLE_INPUTS}' > /root/ansible_inputs.json
+/usr/local/bin/ansible-playbook /root/deployment.yml -e '@/root/ansible_inputs.json'
```

Comment:
There wint be a playbook as you are using the base AMI.

Reply:
The AL2 base AMI was updated to have a deployment.yml as part of the CloudWatch config.